include/deal.II/fe/mapping_q_generic.h

```diff
@@ -81,6 +81,47 @@ template <int,int> class MappingQ;
  * a manifold object to all cells that can then also be used by the mapping
  * classes for higher order mappings.
  *
+ * <h4>Behavior along curved boundaries and with different manifolds</h4>
+ *
+ * As described above, one often only knows a manifold description of a
+ * surface but not the interior of the computational domain. Thus, a
+ * FlatManifold object will be assigned to the interior entities that
+ * describes a usual planar coordinate system where the additional points of
+ * this class are placed exactly according to a bi-/trilinear mapping. When
+ * combined with a non-flat manifold on the boundary, for example a circle,
+ * the two manifold descriptions are in general incompatible. For example, a
+ * FlatManifold would put an interior point located at some small distance
+ * epsilon away from the boundary along a flat line and thus in general
+ * outside the concave part of a circle. If the polynomial degree of
+ * MappingQGeneric is sufficiently high, the transformation from the reference
+ * cell to such a cell would in general contain inverted regions close to the
+ * boundary.
+ *
+ * In order to avoid this situation, this class applies a smoothing on cells
+ * adjacent to the boundary by using so-called Laplace smoothing. In the
+ * algorithm computing new points, all the entities of the cells are passed
+ * through hierarchically, starting from the lines to the quads and finally
+ * hexes. The elements higher up in the hierarchy that sit on a FlatManifold
+ * will then get their points interpolated from all the surrounding points and
+ * not just the corner points. If only a line is associated to a curved
+ * boundary but the adjacent quad is on a flat manifold, the points inside the
+ * quad will be computed according to the deformed line and thus always result
+ * in a well-defined transformation. If this smoothing is undesired, the
+ * optional argument @p smooth_support_points_on_flat_manifold can be used to
+ * disable the smoothing, placing the additional points strictly according to
+ * the manifold.
+ *
+ * While this smoothing approach works well for relatively convergence orders
```

Comment:
I think this is a typo: I don't understand the phrase "for relatively convergence orders".